packages/communication-react/review/communication-react.api.md

```diff
@@ -249,7 +254,7 @@ export interface CallAgentState {
 
 // @beta
 export interface CallAndChatLocator {
-    callLocator: GroupCallLocator;
+    callLocator: GroupCallLocator | /* @conditional-compile-remove-from(stable) TEAMS_ADHOC_CALLING */ CallParticipantLocator;
```

Comment:
Looks like `CallAndChatLocator` is already marked `@beta`, do you need to add this type conditionally?

Reply:
Currently we don't do any beta-type trimming (https://api-extractor.com/pages/configs/api-extractor_json/#dtsrollupbetatrimmedfilepath) so our stable has all the beta types.

Do we want to conditionally compile out types, or allow them but have the beta trimming remove them?
cc @PorterNan

Reply:
Yeah, different feature (`CallAndChatLocator` is for meetings, but `CallParticipantLocator` is for adhoc calls)

Reply:
![image](https://user-images.githubusercontent.com/11863655/153055742-811ecb81-1f2f-4b25-86b5-7b51c51d2d9e.png)
This is how it get tripping out (conditional compile won't work with incomplete sentences/expressions), it just keeps the | operator

Reply:
How would this look like when it is in stable build? It looks like it is not a complete sentence if we doing conditional compiling like this:
callLocator: GroupCallLocator | [nothing];

Reply:
One thing might be good to have a feedback - @JamesBurnside  @prprabhu-ms, do we have a good way to use complete sentence to strip these out - if not, maybe that is an indicator that, type union would be something we might wanna support

Reply:
This should strip out the `|` as well, @prprabhu-ms just recently added this: https://github.com/Azure/communication-ui-library/pull/1427

Reply:
Ohh, did not know about this - that is great since we are supporting it now!